cypress/integration/taskList.spec.js

```diff
@@ -0,0 +1,55 @@
+describe('The System Intake Form', () => {
```

Comment:
Is this in the right PR? I think this might be for a different story, but I'm unsure

Reply:
Good catch. Definitely something from another branch that snuck in.